### ISSUE TITLE
Create a 'on demand' receipt that can be printed for a patient at any time.

### ISSUE THREAD

**Issue by teryhill** [LibreHealthIO/LibreEHR#445]
Create a 'on demand' receipt that can be printed for a patient at any time. This will be a receipt containing the patient information , visit information and payment information. This will have a patient selection and date range.

**Comment by nileshprasad137** [comment]
@teryhill Is this report needed to be added here?
![rep](https://cloud.githubusercontent.com/assets/16336390/23822667/68e9df62-0677-11e7-8e42-a381599cd598.PNG)

**Comment by rksahu17** [comment]
@teryhill Can you assign this task to me?

**Comment by nileshprasad137** [comment]
@rksahu17 Are you working on this?

**Comment by teryhill** [comment]
@nileshprasad137  The Idea for this is a receipt that the office can run at anytime. It would be an on demand receipt. This can be a cleaner format of the Ledger . If a part of the ledger it would need to need be available in both areas of the ledger print, The patient demographics screen and the reports screen.  May be as just a better print option tan what was originally done on the ledger. Just , in my opinion, needs to be a more professional print out. While you are at it see if you could figure out how to print a statement from the Ledger also.

**Comment by teryhill** [comment]
@nileshprasad137  did that explanation help?

**Comment by nileshprasad137** [comment]
@teryhill  Yes! But what would be the contents of such a report? I have some confusion over this.

**Comment by nileshprasad137** [comment]
@teryhill 

> While you are at it see if you could figure out how to print a statement from the Ledger also.

This hasn't been done yet and i intend to do this as and when I get some time.
It would be great if you can recommend some changes in the format , whenever you get some time.

**Comment by tmccormi** [comment]
Pull reverted., reopened issue
